docker-compose.yml

```diff
@@ -74,6 +74,7 @@ services:
       - --character-set-server=utf8
       - --collation-server=utf8_general_ci
       - --innodb-flush-log-at-trx-commit=0
+      - --innodb_log_file_size=132M
```

Comment:
Since this is a command-line argument, it should use dashes instead of underscores, so it should be:
```
- --innodb-log-file-size=132M
```